android/build.gradle

```diff
@@ -119,10 +118,11 @@ dependencies {
         // would add one automatically, probably a version that is not compatible with
         // used kotlin
         def kotlinReflect_version = project.ext.asyncStorageKtVersion
-        def junit_version = "4.12"
-        def robolectric_version = "4.5.1"
-        def truth_version = "1.1.2"
-        def androidxtest_version = "1.1.0"
+        def junit_version = "4.13.2"
+        def robolectric_version = "4.7.3"
+        def truth_version = "1.1.3"
+        def androidxtest_version = "1.4.0"
+        def androidtest_junit_version = "1.1.3"
```

Comment:
Somewhat unrelated: We should consider restructure this better so Dependabot/Renovate can bump them for us. They're pretty dumb and rely on regex when parsing. We did a similar thing here: https://github.com/microsoft/react-native-test-app/blob/trunk/android/dependencies.gradle

Reply:
Could we do a follow up maybe?

Reply:
Absolutely, didn't mean it for this PR.